INSTALL.rst

```diff
@@ -199,83 +196,47 @@ Unsupported platforms include:
 3. Linux on 64-bit ARM processors (Nvidia Jetson):
 
    - Follow the conversation on
-     `issue 4705 <https://github.com/scikit-image/scikit-image/issues/4705>`_.
+     `issue 4705
+     <https://github.com/scikit-image/scikit-image/issues/4705>`_.
+
+4. Windows 32-bit
 
-Although these platforms lack official support, many of the core
-developers have experience with them and can help with questions.
+   SciPy and some other dependencies no longer provide wheels for win32.
 
 If you want to install on an unsupported platform, try
 `building from source <#building-from-source>`_.
 
-Tell us which other platforms you'd like to see ``scikit-image`` on!
-We are very interested in how ``scikit-image`` gets
-`used <https://github.com/scikit-image/scikit-image/issues/4375>`_.
-
-If you'd like to package ``scikit-image`` for an as-yet-unsupported platform,
-`reach out on GitHub <https://github.com/scikit-image/scikit-image/issues>`_.
+If you'd like to package ``scikit-image`` for an as-yet-unsupported
+platform, reach out on our
+`developer forum <https://discuss.scientific-python.org/c/contributor/skimage>`__.
```

Comment:
People keep correcting me on this, but RST links really should use `__` at the end.

Reply:
https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

Reply:
> https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

Thanks for pointing to this, I had no clue! Did you use dunder here only because there is another `developer forum` target (but no other `reach out on GitHub` target, for instance)?

Reply:
Ah, I juse use `__` by default, because I don't plan to re-use the link. Feel free to fix up however you see fit and push here.

Reply:
I'll use dunder by default from now on too :+1: